examples/basic_mysql_connection/index.js

```diff
@@ -4,17 +4,28 @@ var mysql = require('mysql');
 var config = JSON.parse(fs.readFileSync("config-secret.json"))
 
 var connection = mysql.createConnection({
-	host: config.host,
+	 host: config.host,
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
+					'SELECT * FROM todos WHERE Done=1',
+					'SELECT * FROM todos WHERE Done=0',
+					'SELECT * FROM todos GROUP BY Due DESC',
```

Comment:
I don't think this will return what the question asked for: "Get all the todo items, sorted with the most recent first"

In this case I meant "most recently added". Sorry for the confusion, I'll fix it.